core/src/test/java/org/apache/iceberg/view/ViewCatalogTests.java

```diff
@@ -225,8 +243,9 @@ public void createViewErrorCases() {
                     .withQuery(trino.dialect(), trino.sql())
                     .withQuery(trino.dialect(), trino.sql())
                     .create())
-        .isInstanceOf(IllegalArgumentException.class)
-        .hasMessage("Invalid view version: Cannot add multiple queries for dialect trino");
+        .isInstanceOf(Exception.class)
```

Comment:
I don't think this is correct. We should never allow ANY exception to be valid. Why is `IllegalArgumentException` not the correct one?

Reply:
I've removed this change for now. We can discuss it in the REST View support PR